docs/read-side.md

```diff
@@ -277,55 +277,22 @@ const appConfig = {
 
```

Comment:
Please make the following changes to the lines above:

Line 4:
`description: The reSolve framework's read side listens to events that the write side produces. Based on these events, the read side updates the read models that provide data to the queries.`

Please change "provide" to "supply".

Line 9:
`The reSolve framework's read side listens to events that the write side produces. Based on **these** events, the read side updates the **Read Models**, and these models provide data to the queries.`

Please change "provide" to "supply".
Also, why is the word "these" in bold font?

Line 13:
`- **[Projection functions](#updating-a-read-model-via-projection-functions)** build a Read Models state based on incoming events.`

Please change "Read Models" to "Read Model's".

Line 16:
`ReSolve also supports **View Models**. A View Model is a Read Model that can be built on the fly, sent to the client and kept there up-to-date. Refer to the [View Model Specifics](#view-model-specifics) section for more information.`

What do you mean by "kept there up-to-date"? For example, do you mean "built on the fly and sent to the client, so that Read Model data is current"?
If not, please clarify.

Line 123:
`ReSolve provides a unified API to manage data in storage (this code works with any supported storage type). **Read Model Adapters** provide the internal logic a Read Model uses to communicate with DBMSs.`

Please rewrite without using "provides" or "provide".

Line 127:
`## Updating a Read Model via Projection Functions`

Can we change "via" to "with"?

Line 192:
`- To create reactive components, whose state is kept up-to date on the client.`

Please change "up-to date" to "up-to-date".

Reply:
Done
>Also, why is the word "these" in bold font?

It must be by mistake, fixed.

>What do you mean by "kept there up-to-date"?

A View Model maintains a WebSocket connection between the client and the server so the client autimatically receives data updates. I rewrote the paragraph as follows:

"A View Model is a Read Model that is built on the fly. A View Model can maintain a WebSocket connection to push data updates to the client."

Reply:
I like your new version. Thank you.